datamodel/changelogs/0001/03_tww_db_dss.sql

```diff
@@ -61,7 +61,8 @@ WITH (
 CREATE SEQUENCE tww_od.seq_txt_symbol_oid INCREMENT 1 MINVALUE 0 MAXVALUE 999999 START 0;
  ALTER TABLE tww_od.txt_symbol ALTER COLUMN obj_id SET DEFAULT tww_sys.generate_oid('tww_od','txt_symbol');
 COMMENT ON COLUMN tww_od.txt_symbol.obj_id IS 'INTERLIS STANDARD OID (with Postfix/Präfix), see www.interlis.ch';
- ALTER TABLE tww_od.txt_symbol ADD COLUMN classname  varchar(50) ;
+ ALTER TABLE tww_od.txt_symbol ADD COLUMN classname text;
+ ALTER TABLE tww_od.txt_symbol ADD CONSTRAINT length_max_50_symbol_classname CHECK(char_length(classname)<=50);
```

Comment:
```suggestion
 ALTER TABLE tww_od.txt_symbol ADD CONSTRAINT classname_max_length CHECK(char_length(classname)<=50);
```
I would avoid extra long names (constraints are defined for tables, so the table name is implicit) and put the column identifier in front

Reply:
I do not have a strong opinion on this except the constraint name should be :

- [ ] **as understandable as possible**
- [ ] **unique** (for views like tww_ws and tww_reaches)
- [ ] **as small as possible**

Reply:
I can use the shortcut of the class to still make it unique. Else we will have for each class  an `identifier_max_length`
So better `dt_identifier_max_length` (instead of drainless_toilett_identifier_max_lenght')

Reply:
@3nids New version pushed - please check

Reply:
They don't need to be unique as they are defined at the table level.

Again, I'd
* drop the prefix
* drop the length info (if the length changes, no need to rename the constraint)
* length_max => max_length

but it's not that big of a deal.

Reply:
From a users perspective it is helpful to understand why you get the error message. Let's go ahead as technically there are no obstacles anymore.